test/palette.js

```diff
@@ -42,9 +43,20 @@ var navy    = {'R':0   , 'G':0   ,'B':128};
 var blue    = {'R':0   , 'G':0   ,'B':255};
 var yellow  = {'R':255 , 'G':255 ,'B':0};
 var gold    = {'R':255 , 'G':215 ,'B':0};
+
+var whiteLab   = color_convert.rgb_to_lab(white);
```

Comment:
A general note: use snake_case instead of camelCase as this is what is used for the rest of the code.